packages/mui-styled-engine-sc/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/styled-engine-sc",
-  "version": "6.0.0-alpha.18",
+  "version": "6.0.0-alpha.19",
```

Comment:
Is this required? Shouldn't we release this one from `next`?

Reply:
Reverted.